lib/Collection.php

```diff
@@ -77,14 +77,30 @@ public function getIterator()
     }
 
     /**
-     * @return Util\AutoPagingIterator An iterator that can be used to iterate
-     *    across all objects across all pages. As page boundaries are
+     * @return \Generator|StripeObject[] A generator that can be used to
```

Comment:
Clunky, but this is apparently the best (or least bad) way of documenting generators: https://stackoverflow.com/a/36952114/5307473

Reply:
Oh crazy. Well thanks for doing the research on that!